src/main/java/org/truffleruby/Main.java

```diff
@@ -61,6 +60,9 @@
 
 public class Main {
 
+    public static final boolean IS_AOT = Boolean.logicalOr(
+            Boolean.getBoolean("com.oracle.graalvm.isaot"),
+            Boolean.getBoolean("com.oracle.truffle.aot"));
```

Comment:
This is not worth the loss in readability and the need to read that many characters.